README.md

```diff
@@ -14,6 +14,14 @@ to convert all files from Git's internal UTF-8 encoding to IBM-1047, you can spe
 ```
 This will result in Git on z/OS tagging all files as IBM-1047 on checkout. 
 
+If you want the working-tree-encoding to apply to the host platform only, then you can use:
+`platform-working-tree-encoding` where platform is substituted with the system name.
```

Comment:
where do i put this? in the .gitattributes file or is it a git config option?

Reply:
Ah. It is really <platform>-working-tree-encoding where you specify the platdoem eg zos correct?

Reply:
I wasn’t reading it carefully enough. Nice!